### ISSUE TITLE
Signature Tool doesn't work inside sub-menu

### ISSUE THREAD

**Issue by ObiFett** [PDFTron/webviewer-ui#304]
Placing the Signature Tool inside the Misc Tools sub menu prevents it from working. The SignatureOverlay never comes up and instead the toolStylePopup is continuously displayed.

**Comment by ZhijieZhang** [comment]
Hi there,

Very sorry for the delay in the reply.

We are able to reproduce the issue and we plan to fix this in 6.0.

What's happening here is that the signature button in the current header is actually a stateful button. The onClick handler for the button will change depends on if we've saved any signatures yet. If there're no saved signatures then clicking on the document will open the SignatureModal. 

If we move the button to the misc tool group, based on the current code structure the button will 
automatically become a tool button and the onClick handler for a tool button is toggling the toolStylePopup.

This has to do with our code structure and to fix this issue some breaking changes may be introduced. In the dev branch we've actually made the overlay be able to contain other types of button. We will give you a further update when the fix comes out.

Thank you very much for your patience!
